demisto_sdk/commands/common/hook_validations/layout.py

```diff
@@ -155,6 +160,14 @@ def is_incident_field_exist(self, id_set_file, is_circle) -> bool:
                 return False
         return True
 
+    def is_id_equals_name(self):
+        """Check whether the playbook ID is equal to its name.
+
+        Returns:
+            bool. Whether the file id equals to its name
+        """
+        return super(LayoutsContainerValidator, self)._is_id_equals_name('layoutscontainer')
```

Comment:
there is no need to pass parameters to the super function in python3:
[PEP](https://www.python.org/dev/peps/pep-3135/#id4)